### DISCUSSION TITLE
Add Bitcoin Core

### DISCUSSION THREAD

**Discussion by RealCocoArdo** [DesobedienteTecnologico/dtails#25] [Ideas]
I would appriciate adding Bitcoin Core.

I know RunCitadel comes with Bitcoin Core, but that comes with alot of other stuff.

**Comment by DesobedienteTecnologico** [comment]
Hi, thanks for your idea.
Let's create an issue for it.

**Comment by DesobedienteTecnologico** [comment]
Issue created: https://github.com/DesobedienteTecnologico/dtails/issues/26
Will close this discussion and we will keep track of this in the issue.
Regards
